website/static/js/lib/ws4redis.js

```diff
@@ -96,26 +83,15 @@ function WS4Redis(options, $) {
     return Math.random() * maxInterval;
   }
 
-  if(this === undefined) {
-    return new WS4Redis(options, $);
-  }
-  if(options.uri === undefined) {
-    throw new Error('No Websocket URI in options');
-  }
-  if($ === undefined) {
-    $ = jQuery;
-  }
-
-  opts = $.extend({heartbeat_msg: null}, options);
-  connect(opts.uri);
+  connect(options.uri);
 
   this.send_message = function(message) {
     ws.send(message);
   };
 
   this.close = function() {
+    // Avoid reconnect
     ws.onclose = function() {};
-    ws.onerror = function() {};
```

Comment:
This "ws.close" is necessary to avoid some bug on chrome with websocket closing who throw an error (it seems that django-websocket use an old implementation of websocket which is buggy on chrome)